site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+
+If you open the application in a webpage it should still look and work the same as before.
+
+{% instruction 'Why don\'t you go ahead and make the same changes to `Worker` and `WorkerContainer`' %}
+
+Next, we'll start to create a theme.
+
+{% section %}
+
+## Creating a Theme
+
+{% task 'Create a theme directory' %}
+
+Lets go ahead and create a `theme` directory under your project `src`. This is where we will store our themes. For the purpose of this tutorial, we will create a `dojo` theme. Create a `theme.ts` under `src/themes/dojo/theme.ts`. This is the `theme` file into which we will import our themed css files and export the keyed theme for consumption by our widgets.
+
+{% instruction 'Theme the worker widget' %}
+
+In order to theme the `worker` widget, we will need to create a `worker.m.css` file within our theme directory and and use it within our `theme.ts` file. The naming here is important as the object key of the exported theme must match the naming of the widget's css file.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,9,11,16 %}
+
+Lets start by creating a red `worker` so that we can see our theme has been applied correctly.
+
+```css
+/* worker.m.css */
+.root {
+	background: red;
+}
+```
+
+In order to apply our theme, we will need to use a `themeInjector`. Alternatively we *could* set `theme` on the top level widget and pass it down via properties to every child widget but this approach is not very elegant. We can create a `themeInjector` using the `registerThemeInjector` function exported by `ThemeableMixin`. So let's go ahead and update our `main.ts` file to import our theme and create a `themeInjector`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' %}
+
+If you open up your web browser now you should see that the `worker` backgrounds are now `red`.
+
+{% instruction 'Use variables and complete the worker theme' %}
+
+The Dojo 2 build system allows us new `css` features such as `css-custom-properties` by using `postCss` to process our `.m.css` files. We can use these to add variables to `worker.m.css` and to complete it's theme.
+Let go ahead and create `themes/dojo/variables.css` (notice that this file does not have a `.m.css` extension as it is not a `css-module` file).
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/variables.css' lang:css %}
+
+We can now use these variables in `worker.m.css` to create our fully themed `worker`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/worker.m.css' lang:css %}
+
+{% section %}
+
+## Theming Dojo widgets
+
+So far in this tutorial we have themed our custom `worker` widget, but how do we target dojo widgets with our theme. To demonstrate this, we will theme the `workerForm` widget as it contains both domNodes and dojo widgets to be themed.
+
+{% instruction 'Lets create `workerForm.m.css` and include it in our theme.ts' %}
```

Comment:
'Lets create `workerForm.m.css` and include it in our theme.ts' -> 'Let's create `workerForm.m.css` and include it in theme.ts'